meeting-minutes/2022-12-06.md

```diff
@@ -0,0 +1,319 @@
+
+garbados: recently wrote an article describing profound implications of these standards
+
+Brooklyn: I'm Brooklyn Zelenka I've been working on capabilities for a couple years, primarily around UCAN. Similar to Alan, we kind of accidentally reinvented this stuff internally at my company, and then dove into Mark's work and have been doing capabilities stuff since!  Super excited to possibly getting all of these systems working together with OCapN!
+
+warner: I'm Brian Warner, I've breen doing several captp things, built one in Python, in Javascript for Agoric I made one called swingset, dont' have handoffs but we do have promise pipelining
+
+DanC: Dan Connolly, I'm very interested in spritely <-> agoric interop, I used to do w3c standards stuff, now doing ocaps at agoric
+
+Dmitri: Hi I'm Dmitri Zagidulin, I work on VCs and DIDs, but also helped with zcap specification / zcap-ld libraries, so I'm a huge fan of cert style ocaps, I'm interestd in ocapn, couldn't quite understand it, but christine said ocapn rules so I want to join and see if I can understand it better
+
+JakobWeisz: I'm a community contributor to SandstormIO, I found I was basically complaining on G+ at the time about the awful state of cloud stuff and wanted to take my apps with me, kentonv said hey I'm working on something, so I've been following sandstorm since there was such a thing.  it takes webapps and lets you easily run them from one platform and ocap security is built in at a core level with capn proto
+
+isd: I'm Ian Denhart, also a sandstorm contributor, also way back wrote a handful of plugins for gnu social that never got merged, but sort of interesting thing from this group is the sandstorm stuff and I'm also maintainer of the Go/Haskell versions of cap'n proto
+
+isd: Jakob alluded to this but it's roughly a captp
+
+isd: I think with that, spritely, agoric, those are the 3 we're looking to reconcile
+
+danc: when I said spritely <-> agoric interop, I should have included cap'n proto.
+
+elplatt: I'm Edward Platt, relatively new to ocap, largely came to it through following spritely, really excited about it, really fits well with many of the things I've been thinking about, really looking forward to being part of the future, learn from all of y'all
+
+jar286: I'm Jonathan Rees, I worked on lisp implementation and standardization back in the 80s, and lisp/scheme in the 90s, and ocap was in parallel, in ignorance with Joule/E, so that was amazing to discover it was all happening at the time.  So I then discovered all the huge problems I cared about were solved, so I went to work at millenial pharmaceuticals, Science Commons, Open Tree of LIfe, and reconnected with Christine since then,
+
+jar286: got re-interested in ocaps.  And yes I was on the W3C TAG for 4 years or so together
```

Comment:
Should be 'on the W3C TAG with Dan C for 4 years'

(because I had said this in response to a prompt from Christine asking me to make the Dan C connection)

Reply:
(and because the upcoming "we" refers to JAR and Dan C, not JAR and Christine)

Reply:
Fixed, thanks!

Reply:
I expect there's a re-review button next to @jar398 's name above; hit that to prompt him to remove his request for changes?

Or @jar398 , go ahead and signal whether the edits satisfy your request?